src/benchmark/tpch_benchmark.cpp

```diff
@@ -134,12 +134,25 @@ int main(int argc, char* argv[]) {
 
   auto context = opossum::BenchmarkRunner::create_context(*config);
 
+  Assert(!use_prepared_statements || !config->validate, "SQLite validation does not work with prepared statements");
+
+  if (config->validate) {
```

Comment:
I know there is an explanatory comment nested within, but I always find hacky blocks like this to be easier recognizable as such if there is a comment directly above them, e.g., `// HACK: Remove TPCH 15 from verification run, because: ...`